sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java

```diff
@@ -154,6 +167,26 @@ static void setExtraAttributes(AbstractTelemetryBuilder telemetryBuilder, Attrib
                         key.substring(LOGBACK_MDC_PREFIX.length()), String.valueOf(value));
                     return;
                 }
+                if (SemanticAttributes.CODE_FILEPATH.getKey().equals(key)) {
```

Comment:
Are extra attributes expected to grow more in the future? If so, this may see better performance using a `Map` instead of a linear comparison

Reply:
this is a good idea, I'll try it out here

Reply:
done!